### ISSUE TITLE
Images must have alternate text

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#330]
- Text within the image "LocalGov IMS Project" needs to be included as alternative text
- Screenshots within the Help section have no alternative text
